react-app/src/native/components/common/TeamPicker.js

```diff
@@ -71,4 +72,31 @@ class TeamPicker extends React.Component {
   }
 }
 
+function getDay(string) {
```

Comment:
Take a look at datetimehelpers.getDayString(dayNumber). This function should be replaceable with it. Either that or this function should at minimum be moved there.